### PR TITLE
Add daily updated CVE PoC repository by Trickest

### PR DESCRIPTION
**Pull request by mihailotomic**

### CODE REVIEW

**Comment by MrLoverLover85** [comment]
Write access

**Review by its0x08 — APPROVED** [review]

**Review by Benson665 — APPROVED** [review]
Quite looking addressive

**Review by Technetium1 — APPROVED** [review]
This is good

**Review by Austindgk23 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ Repository | Description
 [Cryptography](https://github.com/sobolevn/awesome-cryptography) | Cryptography resources and tools
```
[the remark this answers is not in the document]
Comment (al3in1) [reply README.md:65]: ع

**Review by Jaydalla — APPROVED** [review]

**Comment by cat998** [comment]
已收到！

**Comment by yusuf969** [comment]
This is good

**Comment by Rijubadhan** [comment]
What this frurstiballllllllllllllllllllllllllllllllllllllllll
llllllllllllllll                    l                    lllllllllllllllll
                llllllllllllllllllllllllllllllllllllllllll
llllllllllll                        l                         lllllllllll
ll               llllllllllllllllllllllllllllllllllllllllllll            l
llllllllllllllllll        l          llllllllllllllllllllllllllllllll    l

**Comment by Laffing** [comment]
🙃

**Comment by yusuf969** [comment]
python3 Brute_Force.py -f Account_facebook -l

**Review by Chan9390 — APPROVED** [review]

**Comment by Jaydalla** [comment]
No no it’s all good!! I don’t want no problems .Sent from my iPhoneOn Dec 10, 2023, at 12:24 AM, Chandrapal Badshah ***@***.***> wrote:﻿
Merged #137 into master.

**Review by Sxilxrgoon — COMMENTED** [review]
Good shit

**Pull request merged**